data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			`	Her signs flick at you with a gentleness that you are slowly deciding equates to a cheerful tone. "Thank you for all the help you've provided in recovering this salvage. I have discussed it with the other engineers, and we have decided to give you access to our salvage."`
+			`	She leads you over to the warehouse, scans her ID on the console, and taps a few commands. Then she scans your ID card. "There you go, I have updated your access codes," she signs cheerfully. She hands you your ID and pulls you to a door on the far side of the outfitting shop. Beyond the door is a room filled with outfits and weapons that clearly don't fit the same style as the other Remnant technology. You have seen most of the things here on scans and when salvaging equipment from other ships, but never up close.`
+			`	"This section of our warehouse is the salvage area. Everything in here we have we collected in sufficient quantities that it is available for general use."`
+			`	She nods at a rack of Korath Heat Shunts while she explains, then turns back to you.`
+			`	"Of course the really advanced and rare stuff is still going to the labs first, and won't be available until we have learned all we can from it." She looks around proudly. "We don't have the industrial capacity or the numbers of researchers to keep up with the Alphas, the Korath, or whoever else might be out there. But we are very good at figuring out what others have built and learning from it.`
+			`	"Any salvage or unrecognized technology gets scanned in our quarantine area for threats, then looked over to figure out if there is anything new in it. If there is something new, it is shipped to a lab for our researchers to dissect. Anything we have already figured out ends up in here, where a select few captains are welcome to use it as they see fit."`
+			choice
+				`	"Threats?"`
+					goto threats
+				`	"Interesting."`
+			`	"Yes, it is, isn't it? So many new things out there to discover and learn about.`
+				goto end
+			label threats
+			`	"Yes, sometimes salvage that is brought in is dangerous. System Cores come with fleets of micro-bots, for instance. If they aren't de-activated properly, they keep trying to rebuild their surroundings in the form of the last ship they were configured to serve. And while it hasn't happened yet, we are worried that someday we might run into someone who builds traps into their equipment, or uses nanobots that have run amok. We haven't seen any of that yet, but our science-fiction stories are filled with tales of nanotech apocalypses and things like that.`
+			label end
+			`	"Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "Remnant"
```

Comment:
Perhaps this should be named "Korath Salvage" and the Korath Salvage outfitter (which would then no longer need to be in this event, instead being an outfitter on its own that you could put into remnant.txt) could be added to the Remnant planets in this event. That way if the Remnant ever expand to another planet in the Ember Waste, they can have outfitters with only Remnant tech instead of always appearing to bring the Korath tech where ever they go.

Reply:
That would also be clearer, easier to change if the korath ever get changed and allows more flexibility if the remnant start getting other bits of salvage.

Reply:
That's a good idea.

I've added a "Remnant Salvage": to each of the Remnant planets, and the event now adds those outfits to that outfitter, thus keeping the salvaged outfits separate from the main outfitter.